### DISCUSSION TITLE
PAC Admin Copy missing parameter for security group

### DISCUSSION THREAD

**Discussion by Joseph-Duty-VA** [microsoft/powerplatform-build-tools#910] [Ideas]
When running the PAC admin copy command, it clears out the security group.  I would expect it to either a) maintain the existing security group that was in the environment previously, b) use the source's security group, or c) allow the user to select the security group as an input parameter.  To me c is the best option.  I notice that in the create environment command, you can specify the security group, but its missing from the copy command.  
This dicsussion is a request to add the security group as a parameter to the copy command.  There was a similar request buried in the discussion for https://github.com/microsoft/powerplatform-build-tools/discussions/535 - but since it was not the primary topic of the thread, it likely got lost.

**Comment by bryansonofpatt** [comment]
We have a DevOps pipeline that uses the Copy Environment command to refresh a training environment, but it's kind of useless to automate if we have to manually go in and set the Security group. 
![image](https://github.com/user-attachments/assets/2c7d5ed0-e848-4c19-ab26-e2318464779a)

Why is this not a parameter for 'pac admin copy'?

**Reply by jordanabakerafs** [reply]
+1

Similar question.

https://github.com/microsoft/powerplatform-build-tools/discussions/1110

**Reply by jharkins-eig** [reply]
What I'm seeing is that the original group assigned at environment create is no longer a group that can be selected when trying to set the Security group in the UI.
